openedx/openedx-learning
# Modular import/export taxonomy tags

## Description
Implements [import_tags](https://docs.google.com/document/d/13zfsGDfomSTCp_G-_ncevQHAb4Y4UW0d_6N8R2PdHlA/edit#heading=h.bu5slo87hawp) and [export_tags](https://docs.google.com/document/d/13zfsGDfomSTCp_G-_ncevQHAb4Y4UW0d_6N8R2PdHlA/edit#heading=h.u3ydxm3x6l0n) functions

A modular implementation has been followed for both functionalities. You can see more about this [here](https://github.com/openedx/openedx-learning/pull/64/files#diff-5ba76c1e98a2c00334984b6209fa173366c6996a16478f352d21a8f62b9c8189R4)

## Supporting information

Closes https://github.com/openedx/modular-learning/issues/64

## Testing instructions

- Install requirements `make requirements`
- Read the [import/export explanation](https://github.com/openedx/openedx-learning/pull/64/files#diff-5ba76c1e98a2c00334984b6209fa173366c6996a16478f352d21a8f62b9c8189R4)

Manual testing is possible but not very useful, since we don't yet have views for these models and APIs.

But you can check that the added tests cover the expected import actions and export outputs, and that tests are still covering 100% of the openedx_tagging module.
